src/pjpersist/datamanager.py

```diff
@@ -702,6 +716,12 @@ def reset(self):
         # DB updates, not just reset PJDataManager state
         self.abort(None)
 
+    def _acquire(self):
```

Comment:
Any chance to name it `_acquireConnection` ? Eventually the same with `_releaseConnection`

Reply:
Agreed.

Reply:
Should be `_acquire_conn()` and `_release_conn()` though.

Reply:
I'm really confused what naming convention we use in pjpersist. It seems like camelCase, but it is highly inconsistent.

Reply:
renamed to `_acquire_conn` / `_release_conn`